home-boot.html

```diff
@@ -0,0 +1,158 @@
+<!doctype html>
```

Comment:
Did you want to add this or should we remove this file before merging in your changes?

Reply:
Not sure how that pulled into the tree. I'll remove and update.